bibliopixel/drivers/APA102.py

```diff
@@ -1,23 +1,19 @@
-from spi_driver_base import DriverSPIBase, ChannelOrder
+from . spi_driver_base import DriverSPIBase, ChannelOrder
 
 
 class DriverAPA102(DriverSPIBase):
     """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
-        super(DriverAPA102, self).__init__(num, c_order=c_order,
-                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
+    def __init__(self, num, **kwds):
```

Comment:
Not really... was gonna pull this all in a minute but I can always just revert that one bit (and the other driver I think). But if you are in the middle of changing something, I'll wait...